lib/ansible/plugins/lookup/onepassword_doc.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# (c) 2019, David Taylor <djtaylor13@gmail.com> (onepassword.py used as a starting point)
+# (c) 2018, Scott Buchanan <sbuchanan@ri.pn>
+# (c) 2016, Andrew Zenk <azenk@umn.edu> (lastpass.py used as starting point)
+# (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+    lookup: onepassword_doc
+    author:
+      - David Taylor <djtaylor13@gmail.com>
```

Comment:
```suggestion
      - David Taylor (@djtaylor)
```

Reply:
This information is used by Ansibullbot so list should be in format "name (@<github_handle>)"